proposals/NNNN-close-app.md

```diff
@@ -0,0 +1,63 @@
+# CloseApp RPC
+
+* Proposal: [NNNN](NNNN-close-app.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / iOS / Android / RPC ]
+
+## Introduction
+
+This proposal is about adding a new RPC called `CloseApp` which can be used by an app to send itself into `HMI_NONE`.
```

Comment:
Should the RPC be called `CloseApplication` as it is described in the `Proposed solution` section?